server/handlers/awsJobs.js

```diff
@@ -518,35 +516,15 @@ let handlers = {
           },
         },
       )
-
-      res.send({ jobId: mongoJobId })
-
-      const batchJobParams = aws.batch.buildBatchParams(job)
-
-      aws.batch.startBatchJobs(batchJobParams, mongoJobId, err => {
-        if (err) {
-          // This is an unexpected error, probably from batch.
-          console.log(err)
-          // Cleanup the failed to submit job
-          // TODO - Maybe we save the error message into another field for display?
-          c.crn.jobs.updateOne(
-            { _id: mongoJobId },
-            { $set: { 'analysis.status': 'REJECTED' } },
-          )
-          return
-        } else {
-          let jobLog = aws.batch.extractJobLog(job)
-          emitter.emit(
-            events.JOB_STARTED,
-            {
-              job: jobLog,
-              createdDate: job.analysis.created,
-              retry: true,
```

Comment:
This is needed specifically to log retries.

Reply:
fixed with d6c2c84 below. now startAnalysis takes a retry parameter, and emits logs accordingly